downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -260,38 +258,40 @@ Added support to generate Ballerina client and service declarations from Swagger
     ```
 
 #### JSON-to-record converter
+
 - Improved the JSON-to-record converter tool to be more context-aware and generate records with non-conflicting names.
 
 #### Language Server
 
-* Improved the completion and code action support for pulled modules
-* Improved sorting of module-level completion items
-* Used the code action resolve request for compiler plugin code actions
+- Improved the completion and code action support for pulled modules.
+- Improved sorting of module-level completion items.
+- Used the code action resolve request for compiler plugin code actions.
 
 #### OpenAPI Tool
-- Added support for the `additionalProperties` attribute in OpenAPI object schemas. With this support, the generated Ballerina records can be either open or closed records, based on the `additionalProperties` details.
->**Info:** Thereby, some of your already generated open records may change into closed records, when re-generating using 2201.4.0 (and above) versions.
-- Improved support for `nullable:true` property OpenAPI schema, to generate record fields with default values (e.g. `string? name = ();`), instead of making the field both nilable and optional (e.g. `string? name?;`).
-- Changed the default request and response types of the generated Ballerina resource/remote methods from `json` to `http:Request` and `http:Response`, respectively.
 
+- Added support for the `additionalProperties` attribute in OpenAPI object schemas. With this support, the generated Ballerina records can be either open or closed records based on the `additionalProperties` details.
+  >**Info:** Thereby, some of your already generated open records may change into closed records when re-generating using 2201.4.0 (and above) versions.
+- Improved the support for the `nullable:true` property OpenAPI schema to generate record fields with default values (e.g., `string? name = ();`) instead of making the field both nilable and optional (e.g., `string? name?;`).
```

Comment:
```suggestion
- Improved the behaviour for the `nullable:true` property in OpenAPI schemas, to generate record fields with default values (e.g., `string? name = ();`), instead of making the field both nilable and optional (e.g., `string? name?;`).
```